shared/deployments/publish-cd.sh

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-IFS=$'\n'
-APP_CIDs=($(lerna run publish:cd --no-ci | grep -o "Qm[[:alnum:]]\{44\}$"))
-unset IFS
-echo number of apps published: ${#APP_CIDs[*]}
-
-for cid in "${APP_CIDs[@]}"
-do
-  echo propagating $cid ... $'\n'
-  aragon ipfs propagate $cid > /dev/null &
```

Comment:
Seems like this propagation did not make it into the new root `publish:cd` command. Is that ok?

Reply:
as mentioned above: 
>NOTE I have opted not to continue propagating files from CI because it is a time and resource-intensive process, and also because the publish commands in their current form pin files to our VPS. So for CD testing purposes, we just need to redirect the client in the aragon client settings to utilize our VPS to consume our dev builds for testing

Reply:
Great, thank you for the reminder!

So, this means that anyone who tests these CD builds will need to configure their client to point at our IPFS endpoint, or assets will not load properly. Is that correctL

Reply:
yeah. Yalda and Javi already do this, and we also make it a point to pin the latest aragon app files to this node as well